File: Java_Programming/Competative/Assignments/Assignment_1/program4.java
//Accept one number and check is divisble by 5 or not
import java.util.*;

class DivisionX
{
    public boolean Check(int iNo)
    {
        if((iNo % 5) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        boolean bRet = false;

        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        DivisionX dobj = new DivisionX();
    
        bRet = dobj.Check(iValue);

        if(bRet == true)
        {
            System.out.println("Divisible by 5");
        }
        else
        {
            System.out.println("Not divisible by 5");
        }
    }
}

File: Cpp_Programming/Competative/Assignments/Assignment_1/program4.cpp
//Accept one number and check is divisble by 5 or not

#include<iostream>
using namespace std;

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL Check(int iNo)
{
    if((iNo % 5) == 0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    cout<<"Enter the number\n";
    cin>> iValue;

    bRet = Check(iValue);

    if(bRet == TRUE)
    {
        cout<<"Divisible by 5 \n";
    }
    else
    {
        cout<<"Not divisible by 5 \n";
    }

    return 0;
}